frontend/src/pages/home/index.tsx

```diff
@@ -1,13 +1,183 @@
+        <Grid spacing={2} sx={{display: 'flex', flexDirection: 'column'}}>
+          <Grid
+            item
+            xs={12}
+            md={6}
+            sx={{
+              position: 'relative',
+            }}
+          >
+            <Box
+              sx={{
+                position: 'relative',
+                display: 'relative',
+                paddingTop: '60px',
+                width: {},
+              }}
+            >
+              <img
+                src={bgimg}
+                alt="bgimg"
+                style={{
+                  width: '200%',
+                  height: '100%',
+                  borderRadius: '15px',
+                }}
+              />
+              <Box
+                position="absolute"
+                top={0}
+                left={0}
+                width="200%"
+                height="90%"
+                display="flex"
+                alignItems="center"
+                justifyContent="center"
+                sx={{
+                  backgroundColor: 'rgba(0, 0, 0, 0.4)',
+                  paddingTop: '100px',
+                  borderRadius: '62px',
+                  justifyContent: 'center',
+                  alignItems: 'center',
+                  backgroundPosition: 'center',
+                  backgroundSize: 'cover',
+                  marginTop: '60px',
+                }}
+              >
+                <Typography
+                  sx={{
+                    position: 'absolute',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                    top: '50%',
+                    left: '75%',
+                    transform: 'translate(-120%, -70%)',
+                    color: 'white',
+                    fontSize: '4.0rem',
+                    fontWeight: 'bold',
+                    textAlign: 'center',
+                  }}
+                >
+                  Effortless transportation for students
+                </Typography>
+
+                <Button
+                  variant="contained"
+                  color="primary"
+                  sx={{
+                    position: 'absolute',
+                    height: '50px',
+                    fontWeight: 'bold',
+                    width: '220px',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                    top: '50%',
+                    left: '70%',
+                    transform: 'translate(-170%, 250%)',
+                  }}
+                >
+                  See Schedule
+                </Button>
+              </Box>
+            </Box>
+          </Grid>
+          <Typography
+            align="center"
+            sx={{fontSize: '40px', fontWeight: 'bold', paddingTop: '70px'}}
+          >
+            How to Book
+          </Typography>
+        </Grid>
+
+        <Grid item justifyContent="center" marginBottom="8%">
+          <Box
```

Comment:
You are repeating the same CSS styles three times. You can create a single styled component with these styles and import MUI's styled utility to achieve this.